### DISCUSSION TITLE
Exportação do HTML e CSS

### DISCUSSION THREAD

**Discussion by marcios1985** [vanjarosoftware/Vanjaro.Platform#1013] [General]
boa tarde,

estou conhecendo a ferramenta muito boa pra desenvolver sites rápidos. tem como 
**exporta os codigos fontes (html, css, js) ?**  igual o https://grapesjs.com/demo.html.

**Comment by meetmandeep** [comment]
Vanjaro is not just a page builder; it's a content management system and a web application framework. It automatically manages the frontend framework (such as Bootstrap 5) and so forth. You could export the HTML and CSS by saving the page as a template or exporting the page.
